oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -138,6 +138,12 @@ pub struct Runtime {
     next_node_id: AtomicU64,
 }
 
+#[derive(Debug)]
+pub enum ErroneousChannelStatus {
+    HasInvalid(Vec<ChannelReadStatus>),
+    Err(OakStatus),
+}
```

Comment:
Perhaps `impl From<OakStatus>`, if it makes things simpler when handling errors below?